### PR TITLE
[Keymap] Add motform planck keymap (swedish colemak)

### PR DESCRIPTION
**Pull request by motform**
## Description

My keymap for the Planck keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/motform/keymap.c [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _COLEMAK,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
```
Comment (drashna) [inline keyboards/planck/keymaps/motform/keymap.c]: If you're only using the one default layer, and you're not using the backlight, then you can remove this enum, completely, then.

File: keyboards/planck/keymaps/motform/keymap.c [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case COLEMAK:
```
Comment (drashna) [inline keyboards/planck/keymaps/motform/keymap.c]: If you're only using the one base layer, then you don't actually need this keycode, then.

**Comment by motform** [comment]
Big thanks for the help @drashna! I've pushed a revised version.

**Comment by drashna** [comment]
You're very welcome, and thank you!

**Pull request merged**